src/Uno.UI/UI/Xaml/Controls/MenuBar/MenuBar.cs

```diff
@@ -0,0 +1,53 @@
+#pragma warning disable 108 // new keyword hiding
+#pragma warning disable 114 // new keyword hiding
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Markup;
+
+namespace Windows.UI.Xaml.Controls
+{
+	[ContentProperty(Name="Items")]
+	public partial class MenuBar : Control
+	{
+		private Grid m_layoutRoot;
```

Comment:
naming convention

Reply:
This is code coming from the WinUI repository, so it's keeping their naming convention.